### ISSUE TITLE
Lower Limit of Quadrature Mode

### ISSUE THREAD

**Issue by A622266** [hennichodernich/si5351-iio#1]
Thanks for your contribution of this driver.
What is the lower limit of the quadrature mode, or the lowest frequency at which channel 0 and channel 1 are guaranteed to be 90 degrees phase difference?  I think it may be something around 5MHz, and that below this, the two channels are no longer in quadrature.  Do you agree?

**Comment by hennichodernich** [comment]
You're right, it's 600MHz/127 = 4.7MHz. The driver will print a warning to the kernel log when the phase value is capped at 127.
